src/app/navigation/sign-in/sign-in.component.html

```diff
@@ -0,0 +1,48 @@
+<!-- Default form login -->
```

Comment:
I would throw this inside a mdb card view take alook at: https://mdbootstrap.com/docs/angular/components/cards/ 
I would get rid of the image and the title and the body and see how that looks. If we dont like it we can just ulter what you have to fixed width and maybe add some background color